### PR TITLE
added CRUD on section

### PR DESCRIPTION
**Pull request by mkbhattarai**
Added tests.
Checkstyle and FindBugs find nothing.
Added create for section.

### CODE REVIEW

File: src/main/java/unbanner/StudentRepository.java [hunk 1]
```diff
@@ -8,8 +8,10 @@
 
 public interface StudentRepository extends MongoRepository<Student, String> {
 
-  public Student findByFirstName(String firstName);
+  public List<Student> findByFirstName(String firstName);
```
Comment (cblanto7) [inline src/main/java/unbanner/StudentRepository.java]: findByFirstName should not be returning a list

File: src/test/java/unbanner/StudentTest.java [hunk 2]
```diff
@@ -13,6 +15,7 @@
   public void initObjects() {
     bob = new Student("Bob", "Jones", 0); 
     bob.setId("Dummy String Test");
+    bob.setCourses(new ArrayList<Course>());
```
Comment (cblanto7) [inline src/test/java/unbanner/StudentTest.java]: should be using either a dummy or a mock for course objects inside of this student class test.

File: src/main/java/unbanner/StudentRepository.java (hunk 1 above)
Comment (mkbhattarai) [reply src/main/java/unbanner/StudentRepository.java]: only one student has the same first name? 
if we search by first name, it only returns one.
university has many students with same first name

Comment (cblanto7) [reply src/main/java/unbanner/StudentRepository.java]: on second thought, you are correct about this Mahesh. It absolutely should return a list of students.

**Pull request closed without merging**